draft-ietf-taps-interface.md

```diff
@@ -1747,7 +1747,7 @@ Preconnection -> EstablishmentError<reason?>
 Connection Groups can be created using the Clone Action:
 
 ~~~
-Connection := Connection.Clone(framer?)
+Connection := Connection.Clone(framer?, protocolSpecificProperties?)
```

Comment:
```suggestion
Connection := Connection.Clone(framer?, connectionProperties?)
```
I guess these could not only protocol specific properties, but any property that makes sense on a per-stream basis